lib/hanami/cli/generators/gem/app/operation.erb

```diff
@@ -5,5 +5,9 @@ require "dry/operation"
 
 module <%= camelized_app_name %>
   class Operation < Dry::Operation
+    <%- if generate_db? -%>
+    # Provide `transaction do ... end` method for database transactions
+    include Dry::Operation::Extensions::ROM
```

Comment:
Thanks for thinking to include this! ❤️ 

This is definitely good to go in now. We want these operations to support DB transactions natively.

Seeing this inspired me to create an issue to help us eliminate this line of boilerplate before 2.2 final: https://github.com/hanami/hanami/issues/1437